src/amberscript/parser_buffer_test.cc

```diff
@@ -1092,15 +1092,9 @@ TEST_F(AmberScriptParserTest, BufferDataFilePng) {
 
   Parser parser;
   Result r = parser.Parse(in);
-  ASSERT_TRUE(r.IsSuccess()) << r.Error();
-
-  auto script = parser.GetScript();
-  const auto& buffers = script->GetBuffers();
-  ASSERT_EQ(1U, buffers.size());
+  ASSERT_FALSE(r.IsSuccess());
 
-  ASSERT_TRUE(buffers[0] != nullptr);
-  EXPECT_EQ("foo.png", buffers[0]->GetDataFile());
-  EXPECT_EQ(BufferDataFileType::kPng, buffers[0]->GetDataFileType());
+  EXPECT_EQ("1: missing delegate", r.Error());
```

Comment:
These tests fee like they should set the delegate so we get the same result out?

Reply:
Both GetDataFile and GetDataFileType were removed now that the parser can instantly load the buffer contents. I can still try to add a dummy delegate to allow the parsing to pass.